apps/meteor/tests/e2e/omnichannel/omnichannel-livechat.spec.ts

```diff
@@ -93,6 +93,58 @@ test.describe.serial('OC - Livechat', () => {
 	});
 });
 
+test.describe.serial('OC - Livechat - Visitors closing the room is disabled', () => {
+	let poLiveChat: OmnichannelLiveChat;
+	let poHomeOmnichannel: HomeOmnichannel;
+
+	test.beforeAll(async ({ api }) => {
+		const statusCode = (await api.post('/livechat/users/agent', { username: 'user1' })).status();
+		expect(statusCode).toBe(200);
```

Comment:
We don't need to assert it here, let's call only the endpoint and that's it

Reply:
Hmmm, i was copying this from the rest of the tests. imho there's some value on this as the test would fail when this call returns other than 200, so this is a "fail fast" check. not against removing it tho